### ISSUE TITLE
False Positive for Website URL: https://www.microsoft.com/en-us/avere/

### ISSUE THREAD

**Issue by whalasowski** [microsoft/accessibility-insights-web#6313]
## Describe the bug

We are receiving an alert in S360, link here-   https://vnext.s360.msftcloudes.com/blades/globalandtrusted?blade=KPI:b6a943f5-486a-e3bc-8ede-b2b10c7b680c~SLA:2~AssignedTo:All~Forums:All~_loc:GlobalAndTrusted&def=&peopleBasedNodes=mawhite_team;bebehn_team&tile=&global=4:afa5cdcf-b418-42b2-9cb8-116c85f4c6bb

We believed the problem is fixed and when we run Fast Pass, it passes.   The compliance portal still shows that Accessibility failed and I am unable to run a new scan.  https://webcompliance.trafficmanager.net/ComplianceResults/ddd05212-acdc-4603-12d5-08d913a7927b/Avere

**Comment by pownkel** [comment]
Hi @whalasowski, can you share more details about the OS and browser you're using? I ran fastpass automated checks on each of the failing URLs in the consolidated accessibility report, and got results that were consistent with what the compliance portal reported (I'm using Chrome Version 108.0.5359.125 and Windows 11 Build 22621.963).

Second, if you believe the issue is in the scanning service and not the browser extension, this issue would probably belong in the [accessibility-insights-service](https://github.com/microsoft/accessibility-insights-service) repo instead. For questions specific to compliance scanning, you can contact WCP support using the email in the compliance portal's FAQ section (I notice that the links you posted lead to Microsoft-internal sites, and I'm unsure how much of the information there I can safely discuss here.)
